### PR TITLE
Update EG.yml

### PR DESCRIPTION
**Pull request by gabriel-seow**

### CODE REVIEW

File: public/data/travel/EG.yml [hunk 1]
```diff
@@ -24,8 +22,46 @@ sources:
...
+  outbound_source: https://www.iatatravelcentre.com/international-travel-document-news/1580226297.htm
+  interstate_commercial_flights: 'yes'
+  interstate_commercial_flights_source: https://english.alarabiya.net/en/coronavirus/2020/06/05/Coronavirus-Egypt-s-hotels-set-up-on-site-clinics-for-domestic-tourists-
+  interstate_private_land_transportation: 'yes'
+  interstate_private_land_transportation_source: https://eg.usembassy.gov/u-s-citizen-services/covid-19-information/
+  interstate_public_transportation: 'yes'
+  interstate_public_transportation_source: https://eg.usembassy.gov/u-s-citizen-services/covid-19-information/
+  public_transportation_in_region: 'yes'
+  public_transportation_in_region_source: https://english.alarabiya.net/en/business/economy/2020/08/17/Egypt-transport-authorities-raise-fares-on-Cairo-metro
+  mask: 'no'
```
Comment (jannwong) [inline public/data/travel/EG.yml:39]: mask is required based on the text below?

**Review by jannwong — CHANGES_REQUESTED** [review]
inline comment

File: public/data/travel/EG.yml (hunk 1 above)
Comment (gabriel-seow) [reply public/data/travel/EG.yml:39]: nope! It is only advised, that's why egypt now has it's covid cases going up!

**Review by jannwong — APPROVED** [review]

**Pull request merged**